site/docs/numbers/bulkportins.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: bulkportins
+title: Manage bulk port-in orders
+slug: /numbers/guides/bulkportins
+description: How to manage bulk port-in order using the Bandwidth API
+keywords:
+  - bandwidth
+  - bulkportins
+  - portin
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+export const disconnectId = "{bulkOrderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+[The bulkPortins endpoint](https://dev.bandwidth.com/apis/numbers#operation/ListBulkPortins) is used to manage requests to port a diverse collection of Telephone Numbers into the Bandwidth Dashboard. These telephone numbers will be decomposed into a set of individual port-in orders based on port-type, losing carrier, losing RespOrg, etc., making all reasonable attempts to treat the individual port-in requests in a uniform manner. Like many other requests, the bulkPortins endpoint causes the creation of a Bulk Portins order that is used to manage the porting event throughout the lifecycle of the request.
```

Comment:
if you look at `site/docs/messaging/createMessage.mdx` it looks like we can use a relative URL `/apis/numbers...`

Reply:
+1 for the relative URL, this lets the links work under different domains (staging and local sites). Here's a reference to how voice is doing their links. https://github.com/Bandwidth/api-docs/blob/main/site/specs/voice.json#L3343

Reply:
That link is in the open api spec though, so it's using HTML instead of markdown like this file uses.